src/components/ScreenWrapper.tsx

```diff
@@ -1,15 +1,27 @@
 import React, { FC } from "react";
 import { css, StyleSheet } from "aphrodite";
 import { colors } from "styles/palette";
```

Comment:
in the `src/styles` create an` index.ts`  and keep importing things from  `import {  } from "styles`